### ISSUE TITLE
Python RSA Warning

### ISSUE THREAD

**Issue by vsoch** [singularityhub/sregistry#300]
During build we see:
```
YAML retrying sregistry uwsgi coreschema simplejson openapi-codec user-agents backcall xmlsec spython googleapis-common-protos wrapt
ERROR: awscli 1.18.66 has requirement rsa<=3.5.0,>=3.1.2, but you'll have rsa 4.0 which is incompatible.
```
And this issue is documented here: https://github.com/aws/aws-cli/issues/3660. If a user reports issues here, then we possibly need to downgrade to https://pypi.org/project/rsa/3.4.2/ but I would hope that sooner we can have an update re: the issue I linked above.

**Comment by vsoch** [comment]
@Aneoshun was this part of the issue with using an old config?

**Comment by Aneoshun** [comment]
No, I don't think so. The old-config issues were mainly related to the non-definition of `MINIO_BUCKET` and a wrong setting for `MINIO_EXTERNAL_SERVER`. While this error showed up while Pip resolved the requirement.txt file. So I don't think these different errors are related.
